media/stylus/components/wiki/customcss.styl

```diff
@@ -909,23 +909,12 @@ dl {
 .indicatorInHeadline,
 .blockIndicator,
 .overheadIndicator {
-/*box-shadow: 1px 1px 1px #CCC*/
     margin: 0px 0px 1.5em;
     border: 5px solid;
-    padding: 10px 10px 0px 10px;
+    padding: 10px;
```

Comment:
Looks like it's also a problem with `.overheadIndicator` when used in combination with `.standardNoteBlock` from line 979.